src/db/index.js

```diff
@@ -0,0 +1,54 @@
+class DB {
+    key;
+    schema;
+
+    constructor(key, schema) {
+        if (typeof key !== 'string' || typeof schema !== 'object' || key.length === 0 || schema.length === 0) {
+            return new Error('Bad parameter "key" or "schema"')
```

Comment:
`throw` the Error instead of returning it. 

`return new Error()` => `throw new Error()`